test/e2e/app-dir/segment-cache/prefetch-runtime/prefetch-runtime.test.ts

```diff
@@ -116,14 +117,6 @@ describe('runtime prefetching', () => {
 
       await browser.back()
 
-      // Reveal the link to the second page again. It should not be prefetched again
```

Comment:
these reveals aren't necessary anymore, since state is restored, they were already revealed.